spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -95,6 +96,9 @@
 	@Autowired
 	public KafkaListenerEndpointRegistry registry;
 
+	@Autowired
+	private DeDuper deDuper;
```

Comment:
Funny name, like the Russian phase from the street: `supper, puper, duper` :smile: 
But let it be, because I am not English specialist